packages/vega-expression/src/codegen.js

```diff
@@ -10,9 +10,7 @@ function stripQuotes(s) {
     ) ? s.slice(1, -1) : s;
 }
 
-export default function(opt) {
-  opt = opt || {};
-
+export default function(opt = {}) {
```

Comment:
Can you change this to 

```suggestion
export default function(opt = {}) {
```

[random link](https://rgerg)